docker/tests/Dockerfile

```diff
@@ -35,27 +35,26 @@ RUN apt-get update \
         build-essential \
         curl \
         gnupg2 \
+        libpcl-dev \
         lsb-release \
+        tmux \
+        tmuxinator \
         tzdata \
-        wget
+        wget \
+  && rm -rf /var/lib/apt/lists/* \
+  && apt-get -qq clean
```

Comment:
Is it worth it cleaning it here, considering we'll do an update below again?

Reply:
Yes, this removes the contents of the apt cache from the layer, making the docker container much slimmer. Otherwise you just end up with outdated caches in every layer of the container.